modules/installation-configure-proxy.adoc

```diff
@@ -237,6 +237,16 @@ bundle.
 The installation program does not support the proxy `readinessEndpoints` field.
 ====
 
+[NOTE]
+====
+If the installer times out, restart and then complete the deployment by using the wait-for command of the installer. For example:
+
+[source,yaml]
```

Comment:
I _think_ this is a terminal command so it should be [source,terminal]. I'm not sure though and would advise checking with SME. If it is a terminal command I'd also add the $.

Reply:
I have checked with SME. He confirmed it must be a [source,terminal]. I would make the changes.